lib/bambora/v1/payment_resource.rb

```diff
@@ -3,15 +3,24 @@
 module Bambora
   module V1
     class PaymentResource
-      # Summary: Create and modify payments.
-      # Note: The link below links to all apis includding profiles and tokenization. There aren't great docs explaining
-      #       the /payments endpoints alone.
-      # Docs: https://dev.na.bambora.com/docs/references/payment_APIs/
-      #       https://dev.na.bambora.com/docs/references/payment_SDKs/take_payments/?shell#
-      # Endpoint: https://api.na.bambora.com/v1/payments
-      def initialize(client:, sub_path:)
+      def initialize(client:)
         @client = client
-        @sub_path = sub_path
+        @sub_path = '/v1/payments'
```

Comment:
A change that has been queued up for the `Bambora::V1::ProfileResource`. Remove the required `sub_path` initialization variable, as the endpoint is known beforehand, and can therefore be a class-level detail.